spec/controllers/api/v1/book_suggestion_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+describe API::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating a valid book suggestion' do
+      let!(:new_book_suggestion_attributes) { build(:book_suggestion).attributes }
+      it 'creates a new book suggestion' do
+        expect do
+          post :create, params: { book_suggestion: new_book_suggestion_attributes }
+        end.to change { BookSuggestion.count }.by(1)
+      end
+
+      it 'responds with 201 status' do
+        post :create, params: { book_suggestion: new_book_suggestion_attributes }
+        expect(response).to have_http_status(:created)
+      end
+    end
+  end
+end
+
+describe API::V1::BookSuggestionsController, type: :controller do
+  describe 'POST #create' do
+    context 'When creating an invalid book suggestion' do
+      let!(:new_book_suggestion_attributes) { build(:book_suggestion, user: nil).attributes }
```

Comment:
Same as before.

Reply:
done! but I needed to use the merge method with the attributes that are associations because attributes_for does not create the attribute.